doc/langdef.md

```diff
@@ -77,8 +77,8 @@ productions are only used to recognize separate lexical elements and are ignored
 by the grammar.
 
 ```
-IDENT          ::= [_a-zA-Z][_a-zA-Z0-9]* - RESERVED
-LITERAL        ::= INT_LIT | UINT_LIT | FLOAT_LIT | STRING_LIT | BYTES_LIT
+IDENT          ::= [_a-zA-Z][_a-zA-Z0-9]* ~RESERVED
+LITERAL        ::= UINT_LIT | INT_LIT | FLOAT_LIT | BYTES_LIT | STRING_LIT  
```

Comment:
I don't have a strong preference for switching the order as they are should be considered as having equivalent priority within the rule. In practice, ANTLR does tie-breaking based on the order of the lexer rules, so if you want to keep this change, that would be fine with me.

Reply:
SGTM.